src/model_selection/mod.rs

```diff
@@ -107,3 +107,43 @@ mod tests {
         assert_eq!(x_test.shape().0, y_test.len());
     }
 }
+
+
+///
+/// KFold Cross-Validation
+///
+/// Entities involved in the KFold procedure:
```

Comment:
Will our user benefit from knowing the list of entities in the KFold procedure? 

Doc strings that start from `///` will be displayed in https://docs.rs/smartcore/0.1.0/smartcore/model_selection/index.html and it is important to keep description of the methods and classes short but informative. The danger of keeping a lot of unnecessary information here is that this information might get out-of-sync if not properly maintained. Doc strings should describe the function of the method or a class, meaning of all parameters, some technical details like errors and default values and a short example. Any other information that describes algorithm in greater details should go into the user's manual: https://github.com/smartcorelib/smartcorelib.org